display_applications/ucsc/huba.xml

```diff
@@ -0,0 +1,6 @@
+<display id="ucsc_huba" version="1.0.0" name="display at Track Hub UCSC">
+    <link id="main" name="main">
+        <url>https://genome.ucsc.edu/cgi-bin/hgHubConnect?hubUrl=${qp($hub_file.url + '/myHub/hub.txt')}&amp;hgHub_do_firstDb=on&amp;hgHub_do_redirect=on&amp;hgHubConnect.remakeTrackHub=on</url>
```

Comment:
Should this be configurable?
There are many UCSC instances which could be used.